### ISSUE TITLE
Feature request: Tenders 

### ISSUE THREAD

**Issue by thetaung2** [OpenDevelopmentMekong/wp-odm_theme#945]
Tenders feature - similar to ODC’s “Laws, policies, and agreements" feature
See design mockups 

![tender mockup](https://cloud.githubusercontent.com/assets/20394487/26398104/7a0a24a8-409e-11e7-9034-05f8f33f3376.png)

Screenshot of ODC's Laws page
![laws policies and agreements open development cambodia odc 2017-05-24 16-30-54](https://cloud.githubusercontent.com/assets/20394487/26398111/80c38140-409e-11e7-875e-e357be64ffbf.png)

**Comment by acorbi** [comment]
hi @thetaung2 thanks for raising this.

I see on your mockup that the metadata fields to be shown on the table are:
- title
- buyer
- location
- type
- Opening date
- Closing date
- Status
- Download

These metadata fields are not available on any of the existing CKAN records (dataset, library_records, laws_records or agreements) and therefore we should think about re-using the profiles' codebase for implementing your request, rather than the tabular pages (laws pages) as you propose.

Can you maybe take a look at one of ODC's existing profiles http://opendevelopmentcambodia.net/profiles/ and select one which you find similar to your proposal. Then, it would be very constructive if you could define the elements that you would change, things that you would need to add/remove.

**Comment by prustar** [comment]
@thetaung could you please define "tenders" please? Is this a new documents type?

**Comment by prustar** [comment]
@thetaung could you please define "tenders" please? Is this a new documents type?

**Comment by thetaung2** [comment]
@prustar yes - the Government, Companies and NGOs invite bids for their projects - we are especially interested in documenting the Government tenders.

**Comment by prustar** [comment]
@thetaung2 -- this would then be more appropriate in Agreements than documents. We need to determine the metadata to be collected here and I would prefer not to create a new form for this type of resource. Could you please send an example and we can compare with the existing metadata for each resource to see if there are some parallels that we could reuse.

**Comment by prustar** [comment]
@acorbi -- As you know profile data is not managed well so I think we should use the PostgresQL database directly if possible. This makes it more dynamic.

**Comment by acorbi** [comment]
@prustar the way currently profiles work, involves actually uploading a resource to CKAN that could be managed afterwards with LibreOffice BASE. We should not avoid using profiles, they would be a good option for this use case, IMHO.

**Comment by acorbi** [comment]
@prustar @thetaung2 I suggest having a dedicated call to get on the same page for this implementation

**Comment by thetaung2** [comment]
@acorbi @prustar let me know the days that work for you for the call.
